src/chapters/configuring.xml

```diff
@@ -1672,6 +1672,12 @@ tcp6       0      0 :::4713                 :::*                    LISTEN</comp
                                                 <para>English (UK) keyboard</para>
                                             </listitem>
                                         </varlistentry>
+                                        <varlistentry>
+                                            <term><constant>de-ch-qwertz</constant></term>
+                                            <listitem>
+                                                <para>German (Swiss) keyboard (qwertz)</para>
```

Comment:
I believe the correct name is "Swiss German":

https://en.wikipedia.org/wiki/Swiss_German